README.md

```diff
@@ -1 +1,10 @@
-# HikingJournal
\ No newline at end of file
+# HikingJournal
+
+## Description
+
+HikingJournal is simple journal mobile(only android for now) application providing easy to use way of saving your hiking routes, memorable places location etc. Saved only on your phone.
```

Comment:
Maybe rephrase from
*journal mobile application*
to
*mobile journal application* ?